instrumentation/instabeego/LICENSE.md

```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2023 IBM Corp.
+Copyright (c) 2023 Instana, Inc. https://www.instana.com/
```

Comment:
You can leave only the IBM copyright for new files.
We only keep both for files that already existed before the acquisition  :)

Reply:
Got it. Done